tour_model_eval/first_second_round_evaluation.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "horizontal-transsexual",
+   "metadata": {},
+   "source": [
+    "## This notebook is to show the evaluation (scatter plot) of two rounds of clustering"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "mighty-ukraine",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import emission.core.get_database as edb\n",
+    "import emission.analysis.modelling.tour_model.get_users as gu\n",
+    "import emission.analysis.modelling.tour_model.load_predict as predict\n",
+    "import matplotlib.pyplot as plt\n",
+    "import emission.analysis.modelling.tour_model.get_plot as plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cathedral-pointer",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "participant_uuid_obj = list(edb.get_profile_db().find({\"install_group\": \"participant\"}, {\"user_id\": 1, \"_id\": 0}))\n",
+    "all_users = [u[\"user_id\"] for u in participant_uuid_obj]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "exotic-livestock",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "radius = 100"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "following-fundamentals",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get all/valid user list\n",
+    "user_ls, valid_users = gu.get_user_ls(all_users, radius)\n",
+    "# get all filenames of clustering result\n",
+    "collect_filename = predict.loadModelStage(\"collect_filename\")"
```

Comment:
I don't understand why you need to have the filenames in a list. Can't you just read the file based on the UUID? Can you clarify your design decision here?

Reply:
Oh, I think this one can be deleted. I did use the UUID to save the files. The original design is to save the filename. But it seems not necessary now.